packages/apple-pay-integration/e2e/ApplePaySessionCustomerStepMockObject.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Prefer `eslint-disable-next-line`, as we want to disable the linter on a case-by-case basis, not the entire file.

Reply:
Removed the next line and added it instead as next line